tutorials/mission_specific_analyses/rxte/visualize-rxte-spectra.md

```diff
@@ -0,0 +1,379 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 1.3
+    jupytext_version: 1.17.3
+kernelspec:
+  display_name: heasoft
+  language: python
+  name: heasoft
+title: RXTE Spectral Visualization Example
+date: '2025-09-24'
+authors:
+  - name: Tess Jaffe
+    affiliations:
+      - HEASARC, NASA Goddard
+  - name: David Turner
+    affiliations:
+      - University of Maryland, Baltimore County
+      - HEASARC, NASA Goddard
+---
+
+# Exploring RXTE's spectral observations of Eta Car
+
+## Learning Goals
+
+By the end of this tutorial, you will:
+
+- Know how to find and use observation tables hosted by HEASARC.
+- Be able to search for RXTE observations of a named source.
+- Understand how to retrieve the information necessary to access RXTE spectra stored in the HEASARC S3 bucket.
+- Have the ability to download and visualize those spectra.
+
+
+## Introduction
+This notebook demonstrates an analysis of 16 years of RXTE spectra of Eta Car. 
+
+The RXTE archive contain standard data product that can be used without re-processing the data. These are described in detail in the [RXTE ABC guide](https://heasarc.gsfc.nasa.gov/docs/xte/abc/front_page.html).
+
+We find all the standard spectra, and then use `pyxspec` to load and visualize the data.
+
+:::{important}
+
+**Running On SciServer:**\
```

Comment:
I have a [pending astroquery PR](https://astroquery--3403.org.readthedocs.build/en/3403/heasarc/heasarc.html#links-to-data-products) that will enable the host to be 'guessed', so the calling `download_data` will be sufficient for most cases. I suggest we start using that.